### ISSUE TITLE
standardize the samplesheet for ampliseq to the other nf-core pipelines

### ISSUE THREAD

**Issue by jenmuell** [nf-core/ampliseq#790]
### Description of feature

The samplesheet of ampliseq looks different to the other nf-core pipelines. I would propose the standardization of the headers of the samplesheet in ampliseq for:

- [ ] sampleID to sample
- [ ] forwardReads to fastq_1
- [ ] reverseReads to fastq_2
- [ ] coordinate/adapt the testdata to support the old and new samplesheet layout 

Maybe it would be good to have an adaption from tab to csv as well since with nf-core schema it already supports them.

**Comment by d4straub** [comment]
Thanks!

> Maybe it would be good to have an adaption from tab to csv as well since with nf-core schema it already supports them.

`--input` should already accept tsv, csv, yml as detailed in https://nf-co.re/ampliseq/2.11.0/docs/usage/#samplesheet-input

**Comment by jenmuell** [comment]
Yes it does. I have tested it just to be sure before I implemented the autogeneration of the samplesheet in fetchngs as csv
